tools/ete/ete_init_taxdb.py

```diff
@@ -0,0 +1,7 @@
+import sys
+sys.path.append("./ete/")
```

Comment:
Why is that needed?

Reply:
see my comment below (the official ete code does not work like this)